featuretools/tests/dfs_tests/test_deep_feature_synthesis.py

```diff
@@ -114,6 +114,13 @@ def find_other_agg_features(features):
 
 
 def test_ignores_entities(es):
+    with pytest.raises(TypeError):
+        dfs_obj = DeepFeatureSynthesis(target_entity_id='sessions',
```

Comment:
Since the call to DeepFeatureSynthesis will error it's not necessary to assign a variable to the result